### DISCUSSION TITLE
Integration with AiSuite

### DISCUSSION THREAD

**Discussion by themrzmaster** [langfuse/langfuse#4750] [Ideas]
### Describe the feature or potential improvement

would be really helpful to have a decorator like integration with https://github.com/andrewyng/aisuite
it has simillar API to Openai and the project is growing a lot.

**Comment by marcklingen** [comment]
thanks for sharing! open for contributions on this. generally wrapping the library with the library as shown here should do the trick in the meantime

https://langfuse.com/docs/sdk/python/decorators#log-any-llm-call

**Comment by marcklingen** [comment]
WIP PR for Otel logging in aisuite: https://github.com/andrewyng/aisuite/pull/108
